app/src/main/java/eu/kanade/tachiyomi/ui/reader/loader/ChapterLoader.kt

```diff
@@ -98,6 +104,15 @@ class ChapterLoader(
                 when (format) {
                     is Format.Directory -> DirectoryPageLoader(format.file)
                     is Format.Zip -> ZipPageLoader(tempFileManager.createTempFile(format.file))
+                    is Format.SevenZip -> try {
+                        SevenZipPageLoader(tempFileManager.createTempFile(format.file)) {
```

Comment:
Creating temp files is not a good idea. https://github.com/mihonapp/mihon/issues/134
I would suggest `SevenZipPageLoader` takes a `UniFile` and use the `SeekableByteChannel` overload of `SevenZFile`.